components/brave_wallet/browser/asset_ratio_response_parser.cc

```diff
@@ -422,4 +422,141 @@ mojom::BlockchainTokenPtr ParseTokenInfo(const std::string& json) {
                                      *symbol, decimals, true, "", "");
 }
 
+bool ParseCoinMarkets(const std::string& json,
```

Comment:
i think this should also wait for @spylogsster 's work on rust json validation before being merged. cc @bridiver